doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -41,7 +41,7 @@ For this user guide, the following development kit is required:
 +------------------------+----------+--------------------------------+------------------------------------+
 | **Hardware platforms** | **PCA**  | **Board name**                 | **Build target**                   |
 +========================+==========+================================+====================================+
-| nRF54H20 PDK           | PCA10145 | ``nrf54h20dk_nrf54h20_cpuapp`` | ``nrf54h20dk_nrf54h20_cpuapp@soc1``|
+| nRF54H20 DK            | PCA10175 | ``nrf54h20dk_nrf54h20_cpuapp`` | ``nrf54h20dk_nrf54h20_cpuapp@soc1``|
```

Comment:
I'm guessing it should not be soc1 when building for the DK

Reply:
Removed. Other found occurrences as well.